pkg/auth/server/headers/headers.go

```diff
@@ -0,0 +1,28 @@
+package headers
+
+import (
+	"net/http"
+)
+
+// We cannot set HSTS by default, it has too many drawbacks in environments
+// that use self-signed certs
+const standardHeaders map[string]string{
```

Comment:
I don't know what half of these mean.

Reply:
It means I should commit -a --amend it with the code I had locally, before sending half baked code from when I drafted the first write. sigh.